frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -639,7 +639,7 @@ export const surveyLogic = kea<surveyLogicType>([
                 iteration_count: NEW_SURVEY.iteration_count,
                 iteration_frequency_days: NEW_SURVEY.iteration_frequency_days,
             })
-            actions.setFlagPropertyErrors(NEW_SURVEY.targeting_flag_filters)
+            actions.setFlagPropertyErrors(null)
```

Comment:
what if `NEW_SURVEY.targeting_flag_filters` isn't undefined? will that cause another bug?

Reply:
the problem here is that we're using an `undefined` value for something that expects either an `errors` or `null`

it's also not related to the `targeting_flag_filters` value, because that is defined in the [line above](https://github.com/PostHog/posthog/pull/27705/files/ebfc587bca0fbe9c4d6443002e9a6909dc509ad5#diff-0a051008afcf371e7c0f56a8f251f7d5fd2b20c2c8b491d1b4bd157347e593c9R632)